.changes/trunk-ios-dev.md

```diff
@@ -0,0 +1,6 @@
+---
+"tauri-cli": patch:enhance
+"@tauri-apps/cli": patch:enhance
+---
+
+Set the `TRUNK_SERVE_ADDRESS` environment variable when running on iOS physical devices to support Trunk.
```

Comment:
I think we need to do the same for android, no? see #11137

Reply:
I'll investigate that one separately (need to get an android phone from someone here first)